src/plugins/fakeProfileThemesAndEffects/index.tsx

```diff
@@ -0,0 +1,496 @@
+        user.premiumType = 2;
+    } else if (accent > -1) {
+        user.themeColors = [accent, accent];
+        user.premiumType = 2;
+    }
+}
+
+function updateUserEffectID(user: UserProfile, id: bigint): void {
+    if (id > -1n) {
+        user.profileEffectID = id.toString();
+        user.premiumType = 2;
+    }
+}
+
+const settings = definePluginSettings({
+    prioritizeNitro: {
+        description: "Source to use if profile theme colors / effects are set by both Nitro and About Me",
+        type: OptionType.SELECT,
+        options: [
+            { label: "Nitro", value: true },
+            { label: "About Me", value: false, default: true },
+        ]
+    },
+    "Hide 3y3 Builder": {
+        description: "Hide the 3y3 Builder in the profiles settings page",
+        type: OptionType.BOOLEAN,
+        default: false
+    }
+});
+
+export default definePlugin({
+    name: "FakeProfileThemesAndEffects",
+    description: "Allows profile theming and the usage of profile effects by hiding the colors and effect ID in your About Me using invisible 3y3 encoded characters",
+    authors: [
+        {
+            name: "ryan",
+            id: 479403382994632704n
+        }
+    ],
+    patches: [
+        {
+            find: "UserProfileStore",
+            replacement: {
+                match: /(?<=getUserProfile\(\i\){return )(\i\[\i])/,
+                replace: "$self.decodeUserBio3y3Hook($1)"
+            }
+        },
+        {
+            find: "DefaultCustomizationSections",
+            replacement: {
+                match: /(?<=\.sectionsContainer,children:)\[/,
+                replace: "$&$self.add3y3Builder(),"
+            }
+        }
+    ],
+    settingsAboutComponent: (): JSX.Element => {
+        return (
+            <Forms.FormSection>
+                <Forms.FormTitle tag={"h3"}>{"Usage"}</Forms.FormTitle>
+                <Forms.FormText>
+                    {"After enabling this plugin, you will see custom theme colors and effects in the profiles of other people using this plugin."}
+                    <div className={Margins.top8}>
+                        <b>{"To set your own profile theme colors and effect:"}</b>
+                    </div>
+                    <ol
+                        className={Margins.bottom8}
+                        style={{listStyle: "decimal", paddingLeft: "40px"}}
+                    >
+                        <li>{"Go to your profile settings"}</li>
+                        <li>{"Use the 3y3 Builder to choose your profile theme colors and effect"}</li>
+                        <li>{'Click the "Copy 3y3" button'}</li>
+                        <li>{"Paste the invisible text anywhere in your About Me"}</li>
+                    </ol>
+                </Forms.FormText>
+            </Forms.FormSection>
+        );
+    },
+    settings,
+    decodeUserBio3y3Hook(user: UserProfile | undefined): UserProfile | undefined {
+        if (user !== undefined) {

```

Comment:
```suggestion
        if (user === undefined) return;
```
Do this for a bunch of the other `if` statements, prevents nesting as much and improves readability